auto-multiple-choice: Update get language codes from CPAN

nortcele opened the pull request:
auto-multiple-choice: Update get language codes from CPAN

Get rid of info message:
Locale::Language will be removed from the Perl core distribution in the next major release. Please install it from CPAN.

#### Description

<!-- Note: it is best make pull requests from a branch rather than from master -->
Use CPAN Locale::Codes to suppress info message about Locale::Language removed in future version of Perl core.

###### Type(s)
<!-- update (title contains ": U(u)pdate to"), submission (new Portfile) and CVE Identifiers are auto-detected, replace [ ] with [x] to select -->

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
<!-- Generate version information with this command in shell:
    echo "macOS $(sw_vers -productVersion) $(sw_vers -buildVersion)"; echo "Xcode $(xcodebuild -version | awk '{print $NF}' | tr '\n' ' ')"
-->
macOS 10.14.5 18F132
Xcode 10.2.1 10E1001 

###### Verification <!-- (delete not applicable items) -->
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
<!-- Please don't open a new Trac ticket if you are submitting a pull request. -->
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->

mf2k commented on x11/auto-multiple-choice/Portfile:
```diff
@@ -75,6 +75,7 @@ depends_run             \
                         port:p${perl5.major}-file-mimeinfo \
                         port:p${perl5.major}-glib-object-introspection \
                         port:p${perl5.major}-gtk3 \
+                        port:p${perl5.major}-locale-codes \
```
Since you are adding this to `depends_run`, you need to increase the `revision` by 1 so that users who have it installed will get this new dependency installed.

pmetzger commented:
Why was this closed?

nortcele commented:
I tried for three hours to use git to make the requested modification and ended up with about ten commits in my repository.

At the end, I canceled the pull, deleted my local rep, deleted my fork, created a new fork, cloned it localy, made the modifications again, commit, push, new pull… five minutes.

I've read git doc six month ago but I use it only to push updates to macports once or twice a year so I forgot git to make place in my brain for my main job. This is, for me, the faster way I found to add requested modifications.
